satellite/console/consoleweb/consoleql/mutation.go

```diff
@@ -98,6 +105,38 @@ func rootMutation(log *zap.Logger, service *console.Service, mailService *mailse
 						return nil, err
 					}
 
+					if createUser.PartnerID != "" {
+						offerType = rewards.Partner
+					}
+
+					//TODO: Create a current offer cache to replace database call
+					currentReward, err := service.GetCurrentRewardByType(p.Context, offerType)
+					if err != nil {
+						log.Error("register: failed to get current offer",
+							zap.String("rawSecret", secretInput),
```

Comment:
For security reasons, it's never good idea to log secrets

Reply:
I have seen this to be a pattern in this function. Should we remove the secrets log entirely?

Reply:
No idea if there is a good reason for having them in the logs; maybe those secrets aren't that much important.

Reply:
Now there is a convention for it https://github.com/storj/docs/blob/master/code/Style.md#logging

Please can you open a ticket to report that these new ones must be changed but also the ones that you've seen that made you think that you had to do it here?